Fix mistake in upstream commit

This commit
http://git.gnome.org/browse/xchat-gnome/commit/?id=23aa5dd1a3033c0badd750bedea8ba1b7a677a74
added some "GTK_CHECK_VERSION (2, 23, 0)" check, but it use (2, 16, 0) in some
places, which seems wrong.
--- src/fe-gnome/xtext.c.orig	2011-01-25 18:40:34.000000000 +0800
+++ src/fe-gnome/xtext.c	2011-01-25 18:41:38.000000000 +0800
@@ -1762,7 +1762,7 @@
 	int win_height;
 	int moved;
 
-#if GTK_CHECK_VERSION (2, 16, 0)
+#if GTK_CHECK_VERSION (2, 23, 0)
 	win_height = gdk_window_get_height (GTK_WIDGET (xtext)->window);
 #else
 	gdk_drawable_get_size (GTK_WIDGET (xtext)->window, 0, &win_height);
@@ -3924,7 +3924,7 @@
 	if (line <= RECORD_WRAPS)
 		return ent->wrap_offset[line - 1];
 
-#if GTK_CHECK_VERSION (2, 16, 0)
+#if GTK_CHECK_VERSION (2, 23, 0)
 	win_width = gdk_window_get_width (GTK_WIDGET (xtext)->window);
 #else
 	gdk_drawable_get_size (GTK_WIDGET (xtext)->window, &win_width, 0);
@@ -4755,7 +4755,7 @@
 	int width;
 	int height;
 
-#if GTK_CHECK_VERSION (2, 16, 0)
+#if GTK_CHECK_VERSION (2, 23, 0)
 	width = gdk_window_get_width (GTK_WIDGET (xtext)->window);
 	height = gdk_window_get_height (GTK_WIDGET (xtext)->window);
 #else
